### ISSUE TITLE
release file

### ISSUE THREAD

**Issue by saeohioalpha** [varnishcache/pkg-varnish-cache#144]
I was trying to install Varnish 6.4 on Ubuntu 20.04 per the instructions on
   https://packagecloud.io/varnishcache/varnish64/install#manual-deb
but I get the following back:

> Err https://packagecloud.io/varnishcache/varnish64/ubuntu focal Release
>   404  Not Found [IP: 2600:1f1c:2e5:6901:10ad:ff59:b50e:2773 443]
> E: The repository 'https://packagecloud.io/varnishcache/varnish64/ubuntu focal Release' does not have a Release file.
> E: Failed to download some files
> W: Failed to fetch https://packagecloud.io/varnishcache/varnish64/ubuntu/dists/focal/Release: 404  Not Found [IP: 2600:1f1c:2e5:6901:10ad:ff59:b50e:2773 443]
> E: Some index files failed to download. They have been ignored, or old ones used instead.

It looks like the IP is not found, and also there is no Release file, not sure if that means this is 2 issues or that the Release file will appear once the IP works.

**Comment by gquintard** [comment]
Hi,

`ubuntu:focal` isn't supported for 6.4. More info about packeges here: https://varnish-cache.org/lists/pipermail/varnish-misc/2020-August/026926.html

**Comment by JMC-PK** [comment]
That's a bit hard to follow, but from what I see ubuntu:focal IS supported on packagecloud, but only for varnish releases made after the distribution is released ( 6.4 was released 3/16, but focal was released 4/23, so no Release on package cloud ). Did i get that right?

**Comment by gquintard** [comment]
That's exactly it

**Comment by JMC-PK** [comment]
Cool, thanks for the info!

**Comment by gquintard** [comment]
no worries, I'll put that in the website soon so that you don't have to dig through archives to find the info
